news/2025/2025-02-13-osumania-7k-world-cup-2025-semifinals-recap.md

```diff
@@ -0,0 +1,285 @@
+
+The last gambit from the Filipinos was with their very last pick of the match, being [Xyris - FOXGLOVE](https://osu.ppy.sh/beatmapsets/2318030#mania/4964416) (HB1). A point that was heavily contested by both teams, but that ultimately was won by the Chinese thanks to their first-class consistency! A victory well coveted by team China, who managed to win the match with a 6–4 score and secure themselves a spot on the podium.
+
+### ::{ flag=HK }:: Hong Kong vs. ::{ flag=PE }:: Peru
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/117115848)**\
+**[You can view the match VOD here!](https://www.twitch.tv/videos/2376748946)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/VIDEO_ID?start=XXX&end=XXX" frameborder="0" allowfullscreen></iframe>
+</div>
+
+The last losers' bracket match for Semifinals between Hong Kong (#6) and Peru (#8) had the audience talking about a potential tiebreaker even before the match started, considering the close seed gap between the two teams. With both of them having their respective longest run in 7K MWC history, the face-off determined which team would get eliminated from there on out while the other survives to continue another world cup week in Finals.
+
+Hong Kong protecting [REDALiCE feat. Ayumi Nomiya - Little Star (Full ver.)](https://osu.ppy.sh/beatmapsets/2317896#mania/4964062) (RC2) while banning [Flying Steps - We Are Electric](https://osu.ppy.sh/beatmapsets/2317884#mania/4964044) (RC3) was a rather unexpected decision according to the audience despite being a very common strategy in MWC matches in general. Meanwhile, Peru protected [SiLiS - Hebe](https://osu.ppy.sh/beatmapsets/2317981#mania/4964313) (RC5) and banned [Elliot Hsu - A Guiding Star](https://osu.ppy.sh/beatmapsets/2317897#mania/4964065) (LN1), which was a smart call given that Hong Kong did get a 125,290-point lead against Germany on that map.
+
+Hong Kong opened up the match with their own protect, but it didn't work out as intended as Peru proceeded to get the first break point of the match. With Peru winning their subsequent pick and with the scoreboard being at 0-2 in favor of Peru, Hong Kong tried to pick [seatrus & Halv - The Core](https://osu.ppy.sh/beatmapsets/2317878#mania/4964035) (LN2) thinking it would be the best pick, considering Peru banned LN1. However, the point streak would continue on, tallying the score up to 0-4 in favor of Peru who was on the verge of a match point after winning their protect.
```

Comment:
```suggestion
Hong Kong opened up the match with their own protect, but it didn't work out as intended as Peru proceeded to get the first break point of the match. With Peru winning their subsequent pick and with the scoreboard being at 0–2 in favour of Peru, Hong Kong tried to pick [seatrus & Halv - The Core](https://osu.ppy.sh/beatmapsets/2317878#mania/4964035) (LN2) thinking it would be the best pick, considering Peru banned LN1. However, the point streak would continue on, tallying the score up to 0–4 in favour of Peru who was on the verge of a match point after winning their protect.
```